cronback-services/src/api/auth_middleware.rs

```diff
@@ -200,3 +220,260 @@ pub async fn authenticate<B>(
 
     Ok(resp)
 }
+
+#[cfg(test)]
+mod tests {
+
+    use std::collections::HashSet;
+    use std::fmt::Debug;
+
+    use axum::routing::get;
+    use axum::{middleware, Router};
+    use cronback_api_model::admin::CreateAPIkeyRequest;
+    use hyper::{Body, StatusCode};
+    use tower::ServiceExt;
+
+    use super::*;
+    use crate::api::auth_store::AuthStore;
+    use crate::api::config::ApiSvcConfig;
+    use crate::api::ApiService;
+
+    async fn state() -> AuthenticationState {
```

Comment:
nit: maybe `new_state()` or 'make_state()` to make it clear that this _creates_ a state rather than getting it?